access-control-app/src/cmr/access_control/system.clj

```diff
@@ -31,26 +31,52 @@
   {:default nil
    :parser cfg/maybe-long})
 
+(defconfig log-level
+  "App logging level"
+  {:default "info"})
+
 (defconfig access-control-public-protocol
-  "The protocol to use in documentation examples for the access-control application."
+  "The protocol to use in documentation examples for the access-control application.
```

Comment:
I think I'm going to tweak these again ... to remove the documentation-specific reference, since they are now used for other things, too.